From 25d56bcdfc51809cf0c538721389b9a8cce86eda Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 1 Sep 2023 15:35:33 +0200
Subject: [PATCH 0472/1141] dt-bindings: rng: fix STM32 RNG YAML issues

Fix errors reported by YAML check.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I98abf1a1ff03e8bd50850222f326a4b429e24a1c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/321679
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
---
 Documentation/devicetree/bindings/rng/st,stm32-rng.yaml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml b/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
index 16d7440fb369..59abdc85a9fb 100644
--- a/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
+++ b/Documentation/devicetree/bindings/rng/st,stm32-rng.yaml
@@ -4,7 +4,7 @@
 $id: http://devicetree.org/schemas/rng/st,stm32-rng.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: STMicroelectronics STM32 RNG bindings
+title: STMicroelectronics STM32 RNG
 
 description: |
   The STM32 hardware random number generator is a simple fixed purpose
@@ -15,9 +15,9 @@ maintainers:
 
 properties:
   compatible:
-    oneOf:
-            - const: st,stm32-rng
-            - const: st,stm32mp13-rng
+    enum:
+      - st,stm32-rng
+      - st,stm32mp13-rng
 
   reg:
     maxItems: 1
-- 
2.39.2

